glancesync/glancesync_config.py

```diff
@@ -219,6 +219,16 @@ def __init__(self, configuration_path=None, stream=None, override_d=None):
                 else:
                     target['support_obsolete_images'] = True
 
+                if configparser.has_option(section, 'list_images_timeout'):
+                    target['list_images_timeout'] = configparser.getint(
+                        section, 'list_images_timeout')
+
+                if configparser.has_option(section, 'use_keystone_v3'):
+                    target['use_keystone_v3'] = configparser.getboolean(
+                        section, 'use_keystone_v3')
+                else:
+                    target['use_keystone_v3'] = False
```

Comment:
Only a comment, I think you can specify default required values in the ConfigParser constructor with a dict.